translations/binaries/slps_strings_items_armor.json

```
@@ -929,392 +929,392 @@
             "available_bytes_size": 55,
-            "translation": "은세공으로 장식된 망토입니다.[LINE]\n하얀 빛이 아름답습니다.[LINE]\n[END]"
+            "translation": "은세공으로 장식된 외투.[LINE]\n하얀 빛이 아름답다.[LINE]\n[END]"
         },
         {
             "offset": 1786088,
             "string": "[Sign_Shield]ゴールドクローク[END]",
             "original_bytes_length": 22,
             "available_bytes_size": 23,
-            "translation": "[Sign_Shield]골드 망토[END]"
+            "translation": "[Sign_Shield]골드 클라우크[END]"
         },
         {
             "offset": 1786032,
             "string": "金細工で装飾されたクローク。[LINE]\n別名「豪華なクローク」。[LINE]\n[END]",
             "original_bytes_length": 55,
             "available_bytes_size": 55,
-            "translation": "금세공으로 장식된 망토입니다.[LINE]\n별명은 ‘호화로운 망토’입니다.[LINE]\n[END]"
+            "translation": "금세공으로 장식된 외투.[LINE]\n일명 「호화로운 외투」.[LINE]\n[END]"
         },
         {
             "offset": 1786008,
             "string": "[Sign_Shield]ミスリルクローク[END]",
             "original_bytes_length": 22,
             "available_bytes_size": 23,
-            "translation": "[Sign_Shield]미스릴 망토[END]"
+            "translation": "[Sign_Shield]미스릴 클라우크[END]"
         },
         {
             "offset": 1785936,
             "string": "ミスリル細工装飾されたクローク。[LINE]\nドワーフ族の芸術作品といわれる。[LINE]\n[END]",
             "original_bytes_length": 67,
             "available_bytes_size": 71,
-            "translation": "미스릴 세공으로 장식된 망토입니다.[LINE]\n드워프족의 예술 작품이라고 알려져 있습니다.[LINE]\n[END]"
+            "translation": "미스릴 세공으로 장식된 외투.[LINE]\n드워프족의 예술작품이라고 한다.[LINE]\n[END]"
         },
         {
             "offset": 1785912,
             "string": "[Sign_Shield]メッシュクローク[END]",
             "original_bytes_length": 22,
             "available_bytes_size": 23,
-            "translation": "[Sign_Shield]메쉬 망토[END]"
+            "translation": "[Sign_Shield]메쉬 클라우크[END]"
         },
         {
             "offset": 1785848,
             "string": "網目状に編んだクローク。[LINE]\n通気性が良く、[LINE]\n軽い作りになっている。[END]",
             "original_bytes_length": 63,
             "available_bytes_size": 63,
-            "translation": "그물 모양으로 짜여진 망토입니다.[LINE]\n통기성이 좋고,[LINE]\n가벼운 구조로 되어 있습니다.[END]"
+            "translation": "그물 모양으로 엮은 외투.[LINE]\n통기성이 좋고,[LINE]\n가벼운 구조로 되어 있다.[END]"
         },
         {
             "offset": 1785824,
             "string": "[Sign_Shield]メイジクローク[END]",
             "original_bytes_length": 20,
             "available_bytes_size": 23,
-            "translation": "[Sign_Shield]메이지 망토[END]"
+            "translation": "[Sign_Shield]메이지 클라우크[END]"
         },
         {
             "offset": 1785760,
             "string": "術士のために作られたクローク。[LINE]\n身に付けた者の精神力を高める。[LINE]\n[END]",
             "original_bytes_length": 63,
             "available_bytes_size": 63,
-            "translation": "마법사를 위해 만들어진 망토입니다.[LINE]\n착용자의 정신력을 높여줍니다.[LINE]\n[END]"
+            "translation": "마법사를 위해 만들어진 외투.[LINE]\n착용자의 정신력을 높여준다.[LINE]\n[END]"
```

```suggestion
            "translation": "술사를 위해 만들어진 외투.[LINE]\n착용자의 정신력을 높여준다.[LINE]\n[END]"
```

https://github.com/ToD-DC-Kor/kor-patch/pull/443#issuecomment-2358602133

> 몇몇 아이템에 術士라는 표현이 있는데, 소디언 없으면 정술 못 쓰는 세상에서 術士를 마법사로 번역해도 괜찮은 건지에 대한 고민이 좀 있습니다

정술을 쓰는 것이니까 정술사 또는 술사 정도로 표현하면 어떨까요? 해당 표현이 있는 아이템들은 실제로 술 공격력이나 술 방어력에 영향을 주는 아이템인 것 같으니, 크게 어색하지 않을 것 같습니다.